### ISSUE TITLE
Full history not being added without infile

### ISSUE THREAD

**Issue by joakimglaving** [release-it/conventional-changelog#57]
In the readme it says that `Set a filename as infile to write the changelog to. If this file does not exist yet, it's created with the full history.` However for me that does not happen. I'm not sure if I'm doing something wrong.

After running the release-it command, I can see in the log that the changelog file is created:

```
➜  testing-repo git:(main) ✗ npm run release

> testing-repo@1.1.0 release
> release-it


🚀 Let's release testing-repo (1.1.0...1.2.0)


Changelog:
## [1.2.0](...../testing-repo/compare/v1.1.0...v1.2.0) (2022-10-16)

### Features
....

### Bug Fixes
....

Changeset:
M  .release-it.json
A  CHANGELOG.md
 M package-lock.json
 M package.json

```
However I'm only getting the changelog since the latest tag, not the full history.

Here is my `.release-it.json` config:

```

{
  "git": {
    "commitMessage": "release v${version}",
    "requireCleanWorkingDir": false
  },
  "plugins": {
    "@release-it/conventional-changelog": {
      "preset": "conventionalcommits",
      "infile": "CHANGELOG.md"
    }
  },
  "npm": {
    "publish": false
  }
}

```

Am I missing something super obvious, or is this a bug?

**Comment by agung2001** [comment]
I also had the same problem, not sure what happened.

Other than that I also found another issue if the target file exists, it's not formatted correctly :

`➜  testing-repo git:(main) ✗ npm run release`

this is the result of `changelog.txt`
```
# 📟 Changelog

* fix: problem2 (4861a8c) --> This should have under version [1.2.5] and section Bug Fixes

* fix: problem1 (b38c57b)  --> This should have under version [1.2.4] and section Bug Fixes

## [1.2.3](https://somerepo.com) (2022-12-25)


### Bug Fixes

* fix: problem 0
```

here is my config `.release-it.js`
```
module.exports = {
  "git": {
    "requireBranch": "master",
    "commitMessage": "chore: release v${version}"
  },
  "github": {
    "release": false
  },
  "npm": {
    "publish": false
  },
  "plugins": {
    "@release-it/bumper": {
      "out": ["config.json", "readme.txt", "style.css"]
    },
    "@release-it/conventional-changelog": {
      "filter": function(commit) {
        return ["feat", "fix"].include(commit.type) && commit.scope === 'wp';
      },
      "header": "# 📟 Changelog",
      "infile": "changelog.txt",
      "preset": {
        "name": "conventionalcommits",
        "types": [
          {
            "type": "feat",
            "scope": "wp",
            "section": "Features"
          },
          {
            "type": "fix",
            "scope": "wp",
            "section": "Bug Fixes"
          }
        ]
      }
    }
  }
}
```

**Comment by webpro** [comment]
Thanks for chiming in, @alexanderdavide.

Run with `DEBUG=release-it:* ...` to see debug output and what gets sent to the `conventional-changelog` package.

Going to close this issue due to inactivity. Feel free to re-open.
